code/processes/dqe.q

```diff
@@ -56,6 +56,12 @@ nullchk:{[t;colslist;thres]
   update thresholdfail:nullspercentage>thres from res                                                           /- compare each column's nulls percentage with threshold thres
   }
 
+anomalychk:{[t;colslist;thres]                                                                                  /-function to check percentage of anomalies in each column from colslist of a table t
+  d:({sum[{x~0w}@'x]}each flip tt)*100%count tt:((),colslist)#t;
```

Comment:
what if one of the cols passed is a different type? with the `~` operator, this will only find *long* infinities, is that the desired behaviour?

Reply:
Thanks, we were considering specifically 0w, so I am going to add other infinity types here